### PR TITLE
Add a tooltip for the page title of security hub

### PR DESCRIPTION
**Pull request by AllForNothing**
image

1. This tooltip explains the artifacts total count

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by AllForNothing** [comment]
@Vad1mo We are trying to add a tooltip for the artifact total count:
image
total count = artifacts + accessories + child artifact of index + child artifact of CNAB
Any suggestions for the English text?

**Review by stonezdj — APPROVED** [review]

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 1]
```diff
@@ -1859,6 +1859,7 @@
         "REPO_NAME": "Repository Name",
         "TOOLTIP": "All filters except CVSS3 only support exact matches",
         "NO_VUL": "We could not find any vulnerability",
-        "INVALID_VALUE": "Invalid range"
+        "INVALID_VALUE": "Invalid range",
+        "PAGE_TITLE_TOOLTIP": "The total artifact count includes artifact accessories, child artifacts of the image index, and child artifacts of CNAB artifacts"
```
Comment (Vad1mo) [inline src/portal/src/i18n/lang/fr-fr-lang.json:1863]: The total artifact count also includes artifact accessories, OCI Image Index child artifacts, and CNAB child artifacts

File: src/portal/src/i18n/lang/en-us-lang.json [hunk 2]
```diff
@@ -1893,6 +1893,7 @@
         "REPO_NAME": "Repository Name",
         "TOOLTIP": "All filters except CVSS3 only support exact matches",
         "NO_VUL": "We could not find any vulnerability",
-        "INVALID_VALUE": "Invalid range"
+        "INVALID_VALUE": "Invalid range",
+        "PAGE_TITLE_TOOLTIP": "The total artifact count includes artifact accessories, child artifacts of the image index, and child artifacts of CNAB artifacts"
```
Comment (Vad1mo) [inline src/portal/src/i18n/lang/en-us-lang.json:1897]: The total artifact count also includes OCI artifact accessories, OCI Image Index child artifacts, and CNAB child artifacts

**Review by Vad1mo — CHANGES_REQUESTED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**